crates/encoding/src/lib.rs

```diff
@@ -62,7 +62,8 @@ impl_net_serde!(
     String,
     bytes::Bytes,
     bytestring::ByteString,
-    std::time::Duration
+    std::time::Duration,
+    std::path::PathBuf
```

Comment:
Why do we need to send `PathBuf` over the wire?

Reply:
I think this was left-over from another change that got forgotten during a `histedit` will remove it